doc/api/corepack.md

```diff
@@ -0,0 +1,95 @@
+# Corepack
+
+<!-- introduced_in=REPLACEME -->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
```

Comment:
This needs an extra newline, otherwise the following paragraph gets merged into the blockquote.

```suggestion
> Stability: 1 - Experimental

```